.github/workflows/nodejs.yml

```diff
@@ -56,19 +56,12 @@ jobs:
           echo "isNotEmpty=false" >> $GITHUB_OUTPUT
         fi
 
-    - name: Install LCOV
+    - name: Upload test coverage report
       if: ${{ steps.check-lcov-non-empty.outputs.isNotEmpty == 'true'}}
-      run: |
-        sudo apt-get update && sudo apt-get install --assume-yes lcov
-
-    - name: Report code coverage
-      if: ${{ steps.check-lcov-non-empty.outputs.isNotEmpty == 'true' }}
-      uses: zgosalvez/github-actions-report-lcov@v4
+      uses: codecov/codecov-action@v3.1.5
       with:
-        coverage-files: ./coverage/lcov.info
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        update-comment: true
-
+        files: ./coverage/lcov.info
+        flags: unittests
```

Comment:
What is the fix here?

Reply:
Converting this action over to CodeCov. We have codecov in use on other FINOS repos so in theory, we're bypassing the GitHub permission restrictions. By default, any PR from a forked repository cannot execute write actions against a PR (without contorting the workflow, separating out checkout+test from the coverage reporting which is pretty kludgy). 

I can't really tell from our angle if this is possible though. I need @maoo or @finos-admin to confirm whether the codecov app is available for our repo and if not, do the necessary setup to make it so.

Reply:
@maoo - can we please take a look at the permissions for CodeCov against our repository?

Reply:
> @maoo - can we please take a look at the permissions for CodeCov against our repository?

There was no permission; I've setup codecov and added a `CODECOV_TOKEN` secret on this repo.

Reply:
@coopernetes - awesome, are we good to merge?